src/plugins/messenger/MessengerPreview/components/MessengerImage/components/LightBoxHeader.tsx

```diff
@@ -0,0 +1,97 @@
+import { useMessangerImageContext } from '../MessangerImageContext';
+import { MessagePluginFactoryProps } from "../../../../../../common/interfaces/message-plugin";
+import CloseIcon from '../../../../../../webchat-ui/assets/baseline-close-24px.svg';
+import DownloadIcon from '../../../../../../webchat-ui/assets/baseline-download-24px.svg';
+
+export const getLightBoxHeader = ({ React, styled }: MessagePluginFactoryProps) => {
```

Comment:
When I open the lightbox and tab through it, only the download and close buttons should receive keyboard focus (That is, focus should be trapped within the lightbox). But all the interactive elements in the webchat are receiving focus, which should not be the case.